src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ToModelTransformers.kt

```diff
@@ -757,41 +758,32 @@ fun CaseloadResult.transformToModelFoundProbationRecord(
   isInHardStopPeriod: Boolean,
   isDueForEarlyRelease: Boolean,
   isDueToBeReleasedInTheNextTwoWorkingDays: Boolean,
-): ModelFoundProbationRecord {
-  val hdcad = if (licence.isHdcLicence()) licence.homeDetentionCurfewActualDate else null
```

Comment:
Guess the `hdcad` is not needed anymore? 😅 doesn't look like it was returned but just flagging.